### PR TITLE
Initial Figma-to-code page for Getting Started

### PR DESCRIPTION
**Pull request by rtpascual**
#### Description of changes
Add assets, components, and styling for initial Figma-to-code page. All files are kept separate in order to easily port to other codebase.

Screen Shot 2022-11-03 at 5 43 14 PM

#### Description of how you validated changes

Checked to make sure new page matches UI requirements.

#### Checklist

- [x] PR description included
- [x] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/src/components/Studio/FeatureItem.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  description: string;
+};
+
+export const FeatureItem = ({
+  icon,
+  title,
+  description,
+}: FeatureItemProps): JSX.Element => {
+  return (
+    <Flex gap={35} width={327.75}>
```
Comment (calebpollman) [inline docs/src/components/Studio/FeatureItem.tsx:18]: `327.75` seems maybe large for mobile? Same with `35`

File: docs/src/pages/[platform]/studio/figma.page.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import styles from 'src/styles/studio/figma.module.scss';
+import { useNavigationContext } from 'src/hooks/useNavigationContext';
+import { Button, Image, Text, TextField } from '@aws-amplify/ui-react';
+import { Features } from '@/components/Studio/Features';
+
+const FigmaToCode = () => {
+  const { setAlwaysCollapsible } = useNavigationContext();
+  setAlwaysCollapsible(true);
```
Comment (calebpollman) [inline docs/src/pages/[platform]/studio/figma.page.tsx:10]: Not sure what `setAlwaysCollapsible` is doing, but should it be running on every render?

File: docs/src/pages/[platform]/studio/figma.page.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+  return (
+    <div className={classNames(styles['figma-container'])}>
+      <div className={classNames(styles['figma-landing-hero'])}>
+        <Image src="/studio/figma-to-react.png" alt="Figma to React image" />
+        <Text
+          className={classNames(styles['figma-landing-hero__title'])}
+        >{`Get React code from Figma`}</Text>
```
Comment (calebpollman) [inline docs/src/pages/[platform]/studio/figma.page.tsx:18]: Why the backticks around these strings? Same question for lines 21 and 28

File: docs/src/pages/[platform]/studio/figma.page.tsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+              <Text
+                className={classNames(styles['figma-file-link__text'])}
+              >{`Use your own Figma file`}</Text>
+            }
+            width="100%"
+            placeholder="Figma file link"
+          />
+        </div>
+        <Text className={classNames(styles['figma-landing-hero__text'])}>
+          {'or'}
```
Comment (calebpollman) [inline docs/src/pages/[platform]/studio/figma.page.tsx:35]:
```suggestion
          or
```

File: docs/src/pages/[platform]/studio/figma.page.tsx [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+        </Text>
+        <Button className={classNames(styles['ui-kit'])}>
+          <Text className={classNames(styles['ui-kit__text'])}>
+            {'Use AWS Amplify UI Kit'}
```
Comment (calebpollman) [inline docs/src/pages/[platform]/studio/figma.page.tsx:39]:
```suggestion
            Use AWS Amplify UI Kit
```

Also curious, is "Kit" specific to the Amplify UI in Studio?

File: docs/src/components/Studio/Features.tsx [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+      <Flex>
+        <FeatureItem
+          icon="/studio/configure-form-inputs.svg"
+          title="Configure form inputs"
+          description="Select form inputs to edit, remove, or add in to your existing form"
+        />
+        <FeatureItem
+          icon="/studio/style-your-form.svg"
+          title="Style your form"
+          description="Use colors and spacing yada yada yaday"
```
Comment (calebpollman) [inline docs/src/components/Studio/Features.tsx:19]: Just confirming, is this description value correct?

File: docs/src/pages/[platform]/studio/figma.page.tsx [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+import { Button, Image, Text, TextField } from '@aws-amplify/ui-react';
+import { Features } from '@/components/Studio/Features';
+
+const FigmaToCode = () => {
+  const { setAlwaysCollapsible } = useNavigationContext();
+  setAlwaysCollapsible(true);
+
+  return (
+    <div className={classNames(styles['figma-container'])}>
+      <div className={classNames(styles['figma-landing-hero'])}>
```
Comment (cwoolum) [inline docs/src/pages/[platform]/studio/figma.page.tsx:14]: You don't need `classNames` unless you have more than one class to pass in.

File: docs/src/pages/[platform]/studio/figma.page.tsx [hunk 8]
```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import styles from 'src/styles/studio/figma.module.scss';
```
Comment (cwoolum) [inline docs/src/pages/[platform]/studio/figma.page.tsx:3]: Can this just be a relative path?

File: docs/src/components/Studio/Features.tsx [hunk 9]
```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import { Flex } from '@aws-amplify/ui-react';
+import { FeatureItem } from './FeatureItem';
+import styles from 'src/styles/studio/features.module.scss';
+
+export const Features = () => {
+  return (
+    <Flex className={classNames(styles['feature-container'])}>
+      <Flex>
```
Comment (calebpollman) [inline docs/src/components/Studio/Features.tsx:10]: Just curious, why do we need the `Flex` elements here and on line 22?

File: docs/src/styles/studio/figma.module.scss [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+    color: #232f3e; /* Amazon squid ink */
+  }
+
+  &__text {
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 20px;
+    text-align: center;
+    /* Override dark theme font color */
+    color: #545b64; /* grey 600 */
```
Comment (cwoolum) [inline docs/src/styles/studio/figma.module.scss:38]: Can we pull these colors out into variables? It'll make it easier to use design tokens later.

**Review by calebpollman — COMMENTED** [review]
Don't have much context here but left some questions. Additionally:
- Didn't dig in to the _.scss_ files too much, but are the spacing and font sizes mobile friendly?
- Would be helpful if we had a Amplify hosted demo with the changes

File: docs/src/pages/[platform]/studio/figma.page.tsx (hunk 2 above)
Comment (calebpollman) [reply docs/src/pages/[platform]/studio/figma.page.tsx:10]: But should it be called on every render? And when (if ever) does `alwaysCollapsible` get set back to `false`?

File: docs/src/pages/[platform]/studio/figma.page.tsx (hunk 5 above)
Comment (calebpollman) [reply docs/src/pages/[platform]/studio/figma.page.tsx:39]: Should that be more explicit since Amplify UI is also a library?

**Pull request closed without merging**